package.json

```diff
@@ -1,22 +1,42 @@
 {
 	"name": "@webtides/element-js",
-	"version": "0.3.1",
+	"version": "0.3.2-rc.4",
 	"module": "index.js",
+	"unpkg": "dist/index.js",
```

Comment:
@eddyloewen Would it make sense to add a "main" entry point and point it to the "dist/index.js"?

Reply:
@lukas-schardt Yeah! You're right! It also works with the main field. 👍